pkg/util/fxutil/provide_comp_test.go

```diff
@@ -49,7 +49,7 @@ func TestInvalidArgumentOrReturnValue(t *testing.T) {
 	assertIsSingleError(t, errOpt, "argument must be a function with 0 or 1 arguments, and 1 or 2 return values")
 
 	errOpt = ProvideComponentConstructor(func(FirstComp) SecondComp { return &secondImpl{} })
-	assertIsSingleError(t, errOpt, `constructor must either take 0 arguments, or 1 "requires" struct`)
+	assertIsSingleError(t, errOpt, `must either take 0 arguments, or 1 "requires" struct`)
```

Comment:
Should we assert that the line and file are part of the error message?

Reply:
Thanks for suggesting this. I had thought it would be too brittle to add, but found a way to do it that ended up improving the error message for another possible case.